app/src/main/java/com/example/garden_planner/models/PlantInBed.java

```diff
@@ -1,24 +1,46 @@
 package com.example.garden_planner.models;
 
+import com.example.garden_planner.GardenMethodHelper;
 import com.parse.ParseClassName;
 import com.parse.ParseObject;
 
+import java.time.LocalDate;
 import java.util.Date;
 
 @ParseClassName("PlantInBed")
 public class PlantInBed extends ParseObject {
-    public static final String KEY_DATE = "PlantDate";
+    public static final String KEY_DATE = "whenActuallyPlanted";
     public static final String KEY_TYPE = "PlantType";
     public static final String KEY_NAME = "ThisPlantName";
     public static final String KEY_GARDEN= "plantedInGarden";
+    public static final String KEY_SHOULD_PLANT_DATE = "whenShouldPlant";
+    public static final String KEY_HARVEST_DATE = "whenHarvest";
```

Comment:
I this when it should be harvested, or when it was harvested?

Reply:
it's when it should be harvested - I should make it more clear. In this iteration of the app, it doesn't technically matter when it's harvested since the app doesn't take into account additional harvests, but if in the future I change the model to allow for additional harvests for certain plants, the should be harvested vs. actually harvested distinction would be important.